docs/design/908-proposal-process.md

```diff
@@ -0,0 +1,410 @@
+  consensus has not changed, the proposal is moved to the _Declined_ state.
+
+  If significant discussion happens during the _Likely Decline_ period, the
+  proposal review group may decide to leave the proposal in _Likely Decline_
+  for another week or even move the proposal back to _Active._
+
+  If a proposal is declined, other proposals may be opened to address the same
+  problem with a different approach. Re-opening identical proposals are likely
+  to be set as _Declined as Duplicate_ unless there is significant new
+  information.
+
+* **Declined as Duplicate**: If a proposal is a duplicate of a previously
+  decided proposal, the proposal review group may decline the proposal directly
+  without progressing through other stages.
+
+  Reconsidering a previously declined proposal with the same approach may be
+  considered when there is significant new information.
+
+* **Declined as Infeasible**: If a proposal directly contradicts the core
+  design of Alloy, or if a proposal is impossible to implement efficiently or
+  at all, the proposal review group may decline the proposal as infeasible
+  without progressing through other stages.
+
+  If it seems like there is still general interest from others, or that
+  discussion may lead to a feasible proposal, the proposal may be kept open and
+  the discussion continued.
+
+* **Declined as Retracted**: If a proposal is closed or retracted in a comment
+  by the original author, the proposal review group may decline the proposal as
+  retracted without progressing through other stages.
+
+  If the proposal is retracted in a comment and there is still general interest
+  from others, the proposal may be kept open and the discussion continued.
+
+* **Declined as Obsolete**: If a proposal is obsoleted by changes to Alloy that
+  have been made since the proposal was filed, the proposal review group may
+  decline the proposal as obsolete without progressing through other stages.
+
+  If it seems like there is still general interest from others, or that
+  discussion may lead to a different, non-obsolete proposal, the proposal may
+  be kept open and the discussion continued.
+
+* **Hold**: If a discussion of a proposal requires design revisions or
+  additional information that will not be available for a couple of weeks or
+  more, the proposal review group moves the proposal to the _Hold_ state with a
+  note of what it is waiting on. Once the proposal is unblocked, the proposal
+  can be moved back to the _Active_ state for consideration during the next
+  proposal review.
+
+A publicly available GitHub project tracks proposals and their states.
+
+[governance]: https://github.com/grafana/alloy/blob/main/GOVERNANCE.md
+
+
+### Consensus and disagreement
+
+The goal of the proposal process is to reach general public consensus about the
+outcome in a timely manner.
+
+If the proposal review process cannot identify a general consensus of the
```

Comment:
I am guessing that you mean some kind of weak consensus where one person disagreeing will not necessarily mean that the proposal was veto'd. But I think we should probably be more clear about this, as general consensus may be understood as everyone agreeing.